cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -364,20 +473,23 @@ impl LockedManifestCatalog {
     pub fn list_packages(&self, system: &System) -> Vec<InstalledPackage> {
         self.packages
             .iter()
-            .filter(|package| &package.system == system)
+            .filter(|package| package.system() == system)
             .cloned()
-            .map(|package| InstalledPackage {
-                install_id: package.install_id,
-                rel_path: package.attr_path,
-                info: PackageInfo {
-                    description: package.description,
-                    broken: package.broken,
-                    license: package.license,
-                    pname: package.pname,
-                    unfree: package.unfree,
-                    version: Some(package.version),
+            .map(|package| match package {
+                LockedPackage::Catalog(pkg) => InstalledPackage {
+                    install_id: pkg.install_id,
+                    rel_path: pkg.attr_path,
+                    info: PackageInfo {
+                        description: pkg.description,
+                        broken: pkg.broken,
+                        license: pkg.license,
+                        pname: pkg.pname,
+                        unfree: pkg.unfree,
+                        version: Some(pkg.version),
+                    },
+                    priority: Some(pkg.priority),
                 },
-                priority: Some(package.priority),
+                LockedPackage::Flake(_) => todo!(),
```

Comment:
This determines how to _display_ flake installables in flox list, i'm not sure about the exact requrements for that so i left it a `todo!` for now but will eventually either skip flake installables for a start show them as normal packages (which will be confusing) before this pr is merged.

Reply:
Suggestion: skip them for now, changing it is pretty easy once we have design input.

Reply:
🪓 + followup ticket